src/backend/LamaAPI/Application/Photos/Events/PhotosDeletedEventHandler.cs

```diff
@@ -8,18 +8,18 @@
 
 namespace Application.Photos.Events
 {
-    public class PhotoDeletedEventHandler : IIntegrationEventHandler<PhotoDeletedEvent>
+    public class PhotosDeletedEventHandler : IIntegrationEventHandler<PhotosDeletedEvent>
     {
         private readonly IApplicationDbContext _context;
 
-        public PhotoDeletedEventHandler(IApplicationDbContext context)
+        public PhotosDeletedEventHandler(IApplicationDbContext context)
         {
             _context = context;
         }
 
-        public Task Handle(PhotoDeletedEvent integrationEvent)
+        public Task Handle(PhotosDeletedEvent integrationEvent)
         {
-            System.Console.WriteLine($"PHOTO DELETED EVENT {integrationEvent.PhotoId}");
+            System.Console.WriteLine($"PHOTO DELETED EVENT");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4939304)